### DISCUSSION TITLE
Option to include agent name as "header" of messages sent through whatsapp

### DISCUSSION THREAD

**Discussion by WEBudoGT** [chatwoot/chatwoot#5941] [Feature Requests]
Sometimes there are 2 agents replying to the same chat, and the other end doesn't know who's chatting for obvious reasons.
If you could include an option for those chat channels to include the name of the agent as the body header of each message would fix this.

Example:

[John Doe]
This is the body of the message

**Comment by pranavrajs** [comment]
Thanks for raising this. We are tracking it here. https://github.com/chatwoot/chatwoot/issues/5855

**Reply by WEBudoGT** [reply]
Thanks @pranavrajs !
I just added a small extra request for #5855 , and it's to have the option to select if the agents name shows after the message (as signature) or on top of it (as a header).

Cheers!
